tests/sample.env

```diff
@@ -10,3 +10,15 @@ EMBEDDING_TEST_VALUES=["", "", ""]
 VECTOR_DB_TEST_VALUES=["", "", ""]
 OCR_TEST_VALUES=["", ""]
 X2TEXT_TEST_VALUES=["", ""]
+
+READ_FOLDER_PATH=fsspec-test/input
+WRITE_FOLDER_PATH=fsspec-test/output
+RECURSION_FOLDER_PATH=fsspec-test/output/recursion-test
+READ_PDF_FILE=fsspec-test/input/2.pdf
+READ_TEXT_FILE=fsspec-test/input/1.txt
+WRITE_PDF_FILE=fsspec-test/output/2.pdf
+WRITE_TEXT_FILE=fsspec-test/output/1.txt
+TEST_FOLDER=fsspec-test/test-folder
+GCS_BUCKET=fsspec-test/input
+TEXT_CONTENT=Writing directly from string as read_file is not passed
+FILE_STORAGE='{"GOOGLE_APPLICATION_CREDENTIALS": "/../google-creds.json","PROJECT_ID": "abcdef"}'
```

Comment:
@gaya3-zipstack Like discussed, changing the env to `FILE_STORAGE_PROVIDER` & `FILE_STORAGE_CREDENTIAL` in OSS, so that we can identify the provider used. Adding it here for tracking.
CC : @hari-kuriakose

Reply:
Ok @harini-venkataraman As I mentioned, this is only for unit tests cases that I have. So it should be ok to leave this unchanged.

Reply:
Choosing the provider as discussed can be kept in core for easy use across services as anyway the file storage variants like Permanent and Temporary are kept in core.